### PR TITLE
arc-series can receive classNames from RadialChart data

### PR DESCRIPTION
**Pull request by andrewrocco**
Addresses #444 

Within a `RadialChart`, a custom className for an arc can be passed in through `data`.

```jsx
<RadialChart
  data={[
    {angle: 2, className: 'custom-class'},
    {angle: 1}
  ]}
/>
```

### CODE REVIEW

File: showcase/radial-chart/donut-chart.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default class SimpleRadialChart extends Component {
         innerRadius={100}
         radius={140}
         data={[
-          {angle: 2},
+          {angle: 2, className: 'custom-class'},
```
Comment (mcnuttandrew) [inline showcase/radial-chart/donut-chart.js]: Can you add a test for this?

File: src/plot/series/arc-series.js [hunk 2]
```diff
@@ -149,6 +149,7 @@ class ArcSeries extends AbstractSeries {
           };
           const arcedData = arcBuilder();
           const rowStyle = row.style || {};
+          const seriesClassName = row.className || '';
```
Comment (mcnuttandrew) [inline src/plot/series/arc-series.js]: Nit: would probably call this rowClassName

**Comment by mcnuttandrew** [comment]
If you've already signed the cla you might need to push up one last little commit to get the cla bot to approve

**Comment by andrewrocco** [comment]
@mcnuttandrew It should be good to go!

**Comment by mcnuttandrew** [comment]
@apercu mind cutting a patch?

**Pull request merged**